build/python-client/Dockerfile

```diff
@@ -6,4 +6,14 @@ RUN mkdir /proto
 
 RUN pip install grpcio grpcio-tools poetry
 
+WORKDIR /code
```

Comment:
Once you get this PR synced with main, you should refactor this to no longer use poetry.  

I think `pip install .` should do the trick to get this code installed in the docker container.

Reply:
I have done this sync